client/landing/stepper/declarative-flow/blog-onboarding-translations.js

```diff
@@ -0,0 +1,49 @@
+import { translate } from 'i18n-calypso';
+import { createElement } from 'react';
+
+/* Binary selection page */
+translate( 'Start writing' );
+translate( 'Pick a design first' );
+
+/* Launchpad page */
+translate( "Your blog's almost ready." );
+translate( "Your blog's ready to launch!" );
+translate( 'Keep the momentum going with these final steps.' );
+translate( 'Write your first post' );
+translate( 'Set up your blog' );
+translate( 'Launch your blog' );
+translate( 'Checkout and launch' );
+
+/* Setup blog page */
+translate( 'Make it yours.' );
+translate( 'Give your blog a name' );
+translate( 'A catchy name to make your blog memorable' );
+translate( 'Add a brief description' );
+translate( "Let people know what your blog's about" );
+translate( 'Save and continue' );
+
+/* Pick a domain page */
+translate(
+	'Help your blog stand out with a custom domain. Not sure yet? {{laterLink}}Decide later.{{/laterLink}}',
```

Comment:
Should we replace {{laterLink}} to {{a}}, or the idea is using `components: { laterLink: Some component link }` when it's ready instead of `components: { a: Some component link }`?

Reply:
Per our Slack convo. Going with a strategy like [this](https://github.com/Automattic/wp-calypso/blob/add/blog-onboarding-translations/client/landing/stepper/declarative-flow/internals/steps-repository/choose-a-domain/index.tsx#L105-L110).